perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -1660,11 +1660,6 @@ private void submitEmbeddedGroupApplications(PerunSession sess, List<Group> grou
 				groupApplication.setExtSourceName(app.getExtSourceName());
 				groupApplication.setExtSourceType(app.getExtSourceType());
 				groupApplication.setCreatedBy("Automatically generated");
-				try {
-					getFormForGroup(group);
-				} catch (FormNotExistsException e) {
-					createApplicationFormInGroup(sess, group);
-				}
```

Comment:
Since this logic with autoregistration was already present in the last release, we need to make sure there would not occur any inconsistencies. If we remove this, it might happen that some group was already added to the list of allowed groups for auto registrations and the form had not yet been created. We should either keep this in the code for a while, or we would have to manually create application forms for such groups.

 I'd suggest keeping this code here with a comment that this could be removed in the future. What do you think?

Reply:
Thanks, I didn't think of that... I agree we can remove it later, I commented it in the code and kept it at the original place also.